src/views/contributor.vue

```diff
@@ -26,40 +26,66 @@ if (contributor.value)
 </script>
 
 <template>
-  <div v-if="contributor" mx-auto mt-5 px-4 container>
-    <div mb-5>
-      <h1 mb-2 text-center text-2xl font-bold v-html="contributor.name" />
+  <div v-if="contributor" mx10>
+    <div flex="~ col items-center">
+      <!-- avatars -->
+      <img
+        :src="`https://avatars.githubusercontent.com/${contributor.github}`"
+        :alt="contributor.name"
+        my5
+        w-100px
+        rd-3
+      />
 
-      <a :href="`https://github.com/${contributor.github}`" target="_blank">
-        <img
-          :src="`https://avatars.githubusercontent.com/${contributor.github}`"
-          :alt="contributor.github"
-          class="w-128px"
-        />
-      </a>
+      <!-- name -->
+      <h1 text-2xl font-bold v-html="contributor.name" />
 
-      <span v-if="contributor.twitter">
-        Twitter:
-        <a :href="`https://twitter.com/${contributor.twitter}`" target="_blank">
-          {{ contributor.twitter }}
-        </a>
-      </span>
+      <!-- title -->
+      <span v-if="contributor.title">{{ contributor.title }}</span>
 
-      <component :is="contributor.default" />
+      <!-- social -->
+      <div>
+        <span v-if="contributor.github">
+          Github:
```

Comment:
Use an icon for social media instead
```suggestion
          GitHub:
```

Reply:
> Use an icon for social media instead

Adjustment completed
![image](https://github.com/opensource-f2f/website/assets/111274524/3c4579a7-1d17-413a-a0d4-89e046c315d4)